tests/test_integration.py

```diff
@@ -0,0 +1,41 @@
+#    Copyright 2020 D-Wave Systems Inc.
+
+#    Licensed under the Apache License, Version 2.0 (the "License")
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+
+#        http: // www.apache.org/licenses/LICENSE-2.0
+
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import subprocess
+import unittest
+import time
+import os
+import sys
+
+project_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+class IntegrationTests(unittest.TestCase):
+
+    def test_sudoku(self):
+        demo_file = os.path.join(project_dir, 'sudoku.py')
+        output = subprocess.check_output([sys.executable, demo_file, "problem.txt"])
+        output=str(output)
+        if os.getenv('DEBUG_OUTPUT'):
+            print("Example output \n"+ output)
+
+
+        with self.subTest(msg="Verify if output contains 'THE SOLUTION IS CORRECT' \n"):
+            self.assertIn("THE SOLUTION IS CORRECT".upper(),output.upper())
```

Comment:
Whitespace around comma ([PEP-8](https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements)).